dlt/destinations/impl/bigquery/sql_client.py

```diff
@@ -273,6 +274,20 @@ def _make_database_exception(cls, ex: Exception) -> Exception:
         # anything else is transient
         return DatabaseTransientException(ex)
 
+    def truncate_tables(self, *tables: str) -> None:
+        """NOTE: We only truncate tables that exist, for auto-detect schema we don't know which tables exist"""
+        statements: List[str] = ["DECLARE table_exists BOOL;"]
```

Comment:
should we do it only when autodetect schema is on?

Reply:
done